metadata_service/__init__.py

```diff
@@ -13,6 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Short description of package"""  # Please adapt to package
+"""GHGA Metadata Service"""
```

Comment:
```suggestion
"""A service that manages and stores study- and dataset-related metadata"""
```
This should contain a short description of the service/package.